### ISSUE TITLE
Overflow content is not keyboard accessible

### ISSUE THREAD

**Issue by jessiehuff** [patternfly/patternfly-react#6252]
**Describe the issue. What is the expected and unexpected behavior?**
Often when we have scrollable content this content is not accessible to a keyboard user without adding a tabIndex. This is currently on the consumer. We can keep this as our solution or we can explore ways to do this effort for the consumer. 

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**
([Example](https://www.patternfly.org/v4/components/sidebar#sticky-panel))
![7EDEAB0F-EDB9-4799-8A96-FD69855A4A16_4_5005_c](https://user-images.githubusercontent.com/28281340/131695233-18e4763a-fdbd-4fd2-a58a-70ab167b52d5.jpeg)

- [x] #7078
- [x] #7079

**Comment by nicolethoen** [comment]
All the sidebar examples are setting overflow and height using a style tag. I think there are two possible solutions:

- add a tabIndex to the sidebar content whenever the overflow style is set causing a scrollbar to appear
- add props so the user can control height/overflow (and not use the style prop) and add a tabindex to the content when values are provided.

**Comment by nicolethoen** [comment]
Potentially we can solve this with more explicit a11y documentation about overflow content in general and on the Page a11y docs (and other relevant components).

**Comment by nicolethoen** [comment]
PageSection `hasOverflowScroll` should add a tabindex to allow for scrolling using keyboard.

**Comment by nicolethoen** [comment]
All tasks in this issue have been completed
